src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerActivityExtensions.cs

```diff
@@ -256,9 +257,16 @@ private static void ProcessJaegerTag(ref TagEnumerationState state, string key,
             {
                 PeerServiceResolver.InspectTag(ref state, key, jaegerTag.VStr);
 
-                if (key == SpanAttributeConstants.StatusCodeKey && jaegerTag.VStr == "Error")
+                if (key == SpanAttributeConstants.StatusCodeKey)
                 {
-                    PooledList<JaegerTag>.Add(ref state.Tags, new JaegerTag("error", JaegerTagType.BOOL, vBool: true));
+                    if (jaegerTag.VStr == "Error")
+                    {
+                        PooledList<JaegerTag>.Add(ref state.Tags, new JaegerTag("error", JaegerTagType.BOOL, vBool: true));
+                    }
+                    else if (jaegerTag.VStr == "Unset")
```

Comment:
Is there a way jaeger can tell if status was Unset or Ok?

Reply:
That's basically what this code is doing. It looks for `key == SpanAttributeConstants.StatusCodeKey` and then `VStr` is the `Status` (Ok, Error, or Unset).